### PR TITLE
Quantile Huber Loss to predict Specific Quantile Value

### LINKED ISSUE #36369: Need for more flexible Loss Function
**System information**
- TensorFlow version (you are using): 1.15
- Are you willing to contribute it (Yes/No): Yes

**Describe the feature and the current behavior/state.**
Since, we know that sometimes in the training process, we need the **MAE and MSE type of behavior** simultaneously. We have this kind of loss function already - **Huber Loss**.  But in some cases, we don't need just one value, instead, we need the interval. We need the loss function which can output two values corresponding to lower and upper bounds of the interval. 
This type of loss function is called **Quantile Huber Loss**, which is taken from [this paper](https://arxiv.org/pdf/1402.4624.pdf). The behavior is shown in below image.
73123322-db0dcf80-3fb4-11ea-852d-9dc672bd0906

**Will this change the current api? How?**
No, it will not change the current api. Instead, it will just add one loss function and will include it's tests.

**Who will benefit with this feature?**
This will be very beneficial for the forecasting and time series. Especially, when the variation is very dynamic, then this loss function helps very much in comparison to existing losses in tf.

### PR DESCRIPTION
**Pull request by ashutosh1919**
Fixes #36369 

We know that we have huber loss function added in keras tf-2 already which can perform both kind of behaviour MSE and MAE depending on the scale of data. In most of the prediction and analysis models, we often do not need just median or mean predicted value, but we also need the specific quantile value of prediction. This is the loss function I am adding which is derived from **huber_loss** here
[https://arxiv.org/pdf/1402.4624.pdf](https://arxiv.org/pdf/1402.4624.pdf) I name it **quantile_huber_loss** and the class which calls it is QuantileHuber. I have given great attention to the detail in implementing the function. function takes two extra arguements delta and quantile, where delta is common as the huber_loss and quantile is the float number between 0 and 1. Equation looks as below.
64808942-dd4b3080-d5b5-11e9-9549-2f98fb18a042

### CODE REVIEW

**Comment by ashutosh1919** [comment]
Already added as Pin Ball Loss in Addons repo. Closing this.
